### PR TITLE
Improved management of mobile controls + made them available on all platforms

### LINKED ISSUE #2027: Touchscreen controls should be available on all platforms
Mobile controls should not be mobile-only; they should work on desktops which support touchscreens as well as the web version, which can be played on mobile.

That one's my fault (like many others); I'm on it.

### PR DESCRIPTION
**Pull request by Semphriss**
Fixes #2027

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -152,10 +152,13 @@ endif(WIN32)
 option(UBUNTU_TOUCH "Compile the project for an Ubuntu Touch target" OFF)
 
 # Mobile builds
+option(ENABLE_TOUCHSCREEN_SUPPORT "Enable on-screen controls and event detection for touchscreen devices" ON)
 if (UBUNTU_TOUCH)
-  option(ENABLE_TOUCHSCREEN_SUPPORT "Enable on-screen controls and event detection for touchscreen devices" ON)
+  option(SHOW_TOUCHSCREEN_CONTROLS "Show mobile controls by default" ON)
+  option(HIDE_NONMOBILE_OPTIONS "Hide options that are impractical on mobile decives (e. g. chainging screen resolution)" ON)
```
Comment (tobbi) [inline CMakeLists.txt:158]: Typo: changing

File: CMakeLists.txt [hunk 2]
```diff
@@ -152,10 +152,13 @@ endif(WIN32)
...
 else()
-  option(ENABLE_TOUCHSCREEN_SUPPORT "Enable on-screen controls and event detection for touchscreen devices" OFF)
+  option(SHOW_TOUCHSCREEN_CONTROLS "Show mobile controls by default" OFF)
+  option(HIDE_NONMOBILE_OPTIONS "Hide options that are impractical on mobile decives (e. g. chainging screen resolution)" OFF)
```
Comment (tobbi) [inline CMakeLists.txt:161]: as above: typo.

**Review by tobbi — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (Narre) [reply CMakeLists.txt]: I also see "decives". Same below.

**Review by tobbi — APPROVED** [review]
Looking good from what I can see.

**Pull request merged**